src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -261,8 +267,12 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.browser.name === 'ios';
   }
 
+  private isIOSWKWebView(): boolean {
+    return this.browser.name === 'ios-webview';
```

Comment:
If there are no WebView specific changes we check for, can we add this check under `isIOSSafari()`? 
We do same check for both of them in `requiresCheckForSdpConnectionAttributes` and `supportsCanvasCapturedStreamPlayback`.

Reply:
done -thanks!